### PR TITLE
Adding changelog, simple contribution workflow, and dependencies updates

### PR DESCRIPTION
**Pull request by jamesiri**
*Description of changes:*
Adding changelog, simple contribution workflow, and dependencies updates

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: browser/components/SimpleContributions/steps/Agreement.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ready: true});
+    }, 1 * 1000);
+  }
+
+  render() {
+    return (
+      <div>
+        <ExtensionPoint ext="simple-contrib-landing-dialog">
```
Comment (jpeddicord) [inline browser/components/SimpleContributions/steps/Agreement.tsx:27]: Honestly, I'd leave this whole section blank or with a "To be filled in by administrator" thing.

Comment (jamesiri) [reply browser/components/SimpleContributions/steps/Agreement.tsx:27]: I wanted to add hooks for easy usage but also have a default dialog that made sense.

**Comment by jamesiri** [comment]
Reopening with the linter fixes.

File: browser/components/SimpleContributions/steps/Agreement.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ ready: true });
+    }, 1 * 1000);
+  }
+
+  render() {
+    return (
+      <div>
+        <ExtensionPoint ext="simple-contrib-landing-dialog">
```
Comment (jpeddicord) [inline browser/components/SimpleContributions/steps/Agreement.tsx:27]: Still probably want to nuke this whole section or put in obvious placeholder text.

"This should not be seem as legal advice or a binding contract." doesn't really apply here, and the note about trying to game the system is implying a lot for a workflow that's not necessarily authoritative outside of where this came from.

File: browser/components/SimpleContributions/steps/ApprovalRouter.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import SurveyStep from '../SurveyStep';
+
+import { config } from '../../../../server/config';
+
+export default class AutoApproved extends SurveyStep<{}> {
```
Comment (jpeddicord) [inline browser/components/SimpleContributions/steps/ApprovalRouter.tsx]: This mighta been my fault: Change this class name to ApprovalRouter. This should be the only location where you need to do that.

File: browser/components/SimpleContributions/steps/ContribType.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    this.props.setStepData({
+      ...this.props.ownData,
+      type,
+    });
+  };
+
+  nextStep = () => {
+    if (
+      this.props.ownData.type === 'feature' ||
+      this.props.ownData.type === 'other'
```
Comment (jpeddicord) [inline browser/components/SimpleContributions/steps/ContribType.tsx]: These almost definitely need to be configurable (wouldn't even leave the default).

Actually, looks like you might have done this in `config.contributions.autoApprove.allowedTypes`; need to update that here too

File: browser/components/SimpleContributions/steps/TicketSubmission.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+export default class TicketSubmission extends SurveyStep<{}> {
+  render() {
+    return (
+      <div>
+        <p>
+          Your change was not able to be automatically approved. You must follow
+          the regular <Link to="/contribute">Log Contribution</Link> process.
+        </p>
+
+        <ExtensionPoint ext="simple-contrib-auto-approve-deny">
```
Comment (jpeddicord) [inline browser/components/SimpleContributions/steps/TicketSubmission.tsx:11]: Move this up a little to include the previous paragraph as well.

File: config/default.js [hunk 6]
```diff
@@ -53,16 +54,53 @@ config.roles = {
...
   ],
   poc: [
-    'ADD A POC', // user(s) that are points of contacts for CCLAs
+    null, // user(s) that are points of contacts for CCLAs
   ],
 };
 
+// Line limit for the diff checker
+config.contributions = {
+  autoApprove: {
```
Comment (jpeddicord) [inline config/default.js:66]: 👍

File: config/default.js [hunk 7]
```diff
@@ -53,16 +54,53 @@ config.roles = {
...
+config.cron = {
+  type: 'onbox', // runs cron-like js scheduler
+  cronTime: '* * * * *', // configure cron schedule
+  timeZone: 'America/Los_Angeles', // configure timezone
+};
+
 // load once asked for
 function load() {
-  return Promise.resolve(config);
+  // verify users filled out the config
```
Comment (jpeddicord) [inline config/default.js:91]: nice

File: browser/components/SimpleContributions/steps/Agreement.tsx (hunk 2 above)
Comment (jamesiri) [reply browser/components/SimpleContributions/steps/Agreement.tsx:27]: Fair enough. I will trim it to be more placeholdery.

File: browser/components/SimpleContributions/steps/ApprovalRouter.tsx (hunk 3 above)
Comment (jamesiri) [reply browser/components/SimpleContributions/steps/ApprovalRouter.tsx]: Changed. This was the only location.

File: browser/components/SimpleContributions/steps/ContribType.tsx (hunk 4 above)
Comment (jamesiri) [reply browser/components/SimpleContributions/steps/ContribType.tsx]: Good catch, fixing.

File: browser/components/SimpleContributions/steps/TicketSubmission.tsx (hunk 5 above)
Comment (jamesiri) [reply browser/components/SimpleContributions/steps/TicketSubmission.tsx:11]: Brought it all together.

**Review by jpeddicord — APPROVED** [review]

**Pull request merged**